build/images/Dockerfile.build.collector

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.15 as go-ipfix-build
+
+WORKDIR /go-ipfix
+
+COPY go.mod /go-ipfix/go.mod
+
+RUN go mod download
+
+COPY . /go-ipfix
+
+RUN make collector
+
+
+FROM antrea/base-ubuntu:2.14.0
+
+FROM ubuntu:18.04
```

Comment:
Just curious about why we want to use both `antrea/base-ubuntu` and `ubuntu`

Reply:
Thank for the question. We do not need antrea base-ubuntu image.